### ISSUE TITLE
transaction type not supported: type 2 rejected, pool not yet in London

### ISSUE THREAD

**Issue by wzhsh90** [ethereum/go-ethereum#30838]
#### System information

geth version 1.13.15-stable-c5ba367e
CL client & version: Clef version 1.14.12-unstable-e26468f6-20241017
github.com/ethereum/go-ethereum version v1.14.12
OS & Version: Windows/Linux/OSX
Commit hash : (if `develop`)

I can not send DynamicFeeTx transaction ,only can send LegacyTx.

**Comment by MariusVanDerWijden** [comment]
Is your node synced? Can you add some more details so we can reproduce it easier?

**Comment by wzhsh90** [comment]
This is a private network, what's information do you need ?


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ethereum/go-ethereum"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年11月29日(星期五) 下午2:44
***@***.***&gt;;
抄送:&nbsp;"@***@***.******@***.***&gt;;
主题:&nbsp;Re: [ethereum/go-ethereum] transaction type not supported: type 2 rejected, pool not yet in London (Issue #30838)





 
Is your node synced? Can you add some more details so we can reproduce it easier?

**Comment by holiman** [comment]
> This is a private network, what's information do you need ?

During startup, geth spits out the chain config - what forks are enabled at what blocks. That's one piece of info we need. 
Also, what block is _your node_ at. That's the second. 

> geth version 1.13.15-stable-c5ba367e

That's the last geth-version supporting pre-merge network. I guess you are using a pre-merge network?

**Comment by MariusVanDerWijden** [comment]
@wzhsh90 Could you send us your chain configuration?

**Comment by wzhsh90** [comment]
{
&nbsp; "config": {
&nbsp; &nbsp; "chainId": 211029,
&nbsp; &nbsp; "homesteadBlock": 0,
&nbsp; &nbsp; "eip150Block": 0,
&nbsp; &nbsp; "eip155Block": 0,
&nbsp; &nbsp; "eip158Block": 0,
&nbsp; &nbsp; "byzantiumBlock": 0,
&nbsp; &nbsp; "constantinopleBlock": 0,
&nbsp; &nbsp; "petersburgBlock": 0,
&nbsp; &nbsp; "istanbulBlock": 0,
&nbsp; &nbsp; "berlinBlock": 0,
&nbsp; &nbsp; "clique": {
&nbsp; &nbsp; &nbsp; "period": 5,
&nbsp; &nbsp; &nbsp; "epoch": 30000
&nbsp; &nbsp; }
&nbsp; },
&nbsp; "difficulty": "1",
&nbsp; "gasLimit": "8000000",
&nbsp; "extradata": "0x0000000000000000000000000000000000000000000000000000000000000000D1499eE78aA4e9c7f4d5De75d2d9d856665763470000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
&nbsp; "alloc": {
&nbsp; &nbsp; "6627a8D651C193Fb26204989069bDFf877EB3483": { "balance": "100000000000000000000000000000" }
&nbsp; }
}



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ethereum/go-ethereum"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月31日(星期二) 晚上6:09
***@***.***&gt;;
抄送:&nbsp;"@***@***.******@***.***&gt;;
主题:&nbsp;Re: [ethereum/go-ethereum] transaction type not supported: type 2 rejected, pool not yet in London (Issue #30838)





 
@wzhsh90 Could you send us your chain configuration?

**Comment by MariusVanDerWijden** [comment]
Yep, as I expected, your chain does not have the london hardfork enabled yet. This means you can not send 1559 transactions. So either you need to fork the chain to the london spec or you need to stop sending 1559 transactions and only send legacy transactions

**Comment by MariusVanDerWijden** [comment]
I will close this for now, there's nothing we can really do here, sorry
